okhttp/src/test/java/io/grpc/okhttp/OkHttpClientTransportTest.java

```diff
@@ -482,7 +483,7 @@ public void windowUpdate() throws Exception {
   public void windowUpdateWithInboundFlowControl() throws Exception {
     initTransport();
     MockStreamListener listener = new MockStreamListener();
-    clientTransport.newStream(method, new Metadata(), listener).request(1);
+    OkHttpClientStream stream = clientTransport.newStream(method, new Metadata(), listener);
```

Comment:
Sorry, replied too hurry.

I think using 3 on the checks is better, it can potentially checks the streams id should be 3. I don't see any benefit of using stream.id() in the test checks.